istioctl/pkg/auth/upgrade.go

```diff
@@ -247,12 +263,12 @@ func (ug *Upgrader) addRoleNameToWorkloadLabelMapping(serviceRolePolicy model.Co
 // mapRoleNameToWorkloadLabels maps the roleName from namespace to workload labels that its rules originally
 // applies to.
 func (ug *Upgrader) mapRoleNameToWorkloadLabels(roleName, namespace, fullServiceName string) error {
-	// TODO(pitlv2109): Handle when services = "*" or wildcards.
-	if strings.Contains(fullServiceName, "*") {
-		return fmt.Errorf("services with wildcard * are not supported")
+	// TODO: Handle suffix for services as described at https://istio.io/docs/reference/config/authorization/istio.rbac.v1alpha1/#AccessRule
```

Comment:
We don't really need to include the URL here.

Reply:
Will remove but just curious, why is it against putting URLs in comments or TODOs? Is it because they could be dead links? I think URLs are pretty helpful for learning about the context if they are alive.